core/src/main/scala/cats/collections/Map.scala

```diff
@@ -7,7 +7,7 @@ import scala.collection.mutable.ListBuffer
 /**
  * A tree based immutable map.
  */
-class AvlMap[K,V](val set: AvlSet[(K,V)]) {
+class AvlMap[K,+V](val set: AvlSet[(K,V)]) {
```

Comment:
I only made this covariant in V. I think that we _could_ make it covariant on `K`, but I don't know if it is useful.

Reply:
Probably not.